src/ontology/mondo-edit.obo

```diff
@@ -497585,7 +497587,7 @@ xref: OMIM:619637 {source="MONDO:equivalentTo"}
 xref: UMLS:C5562029 {source="MEDGEN:1794239", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0044807 {source="OMIM:619637"} ! inherited dystonia
 is_a: MONDO:0100617 {source="https://orcid.org/0000-0001-5208-3432"} ! VPS11-related neurological disorder
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/14583 {source="OMIM:619637"} ! VPS11
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/14583 ! VPS11
```

Comment:
@sabrinatoro the OMIM source was removed in this case because it's marked as provisional (see https://omim.org/entry/619637) and therefore does not meet the conditions to be maintained (see https://github.com/monarch-initiative/omim?tab=readme-ov-file#necessary-conditions-for-disease-defining-associations).

Is there a source you would like me to add?

Reply:
I added the source that is in the issue tracker for this term.